tests/unit/components/search-input/SearchInput.spec.jsx

```diff
@@ -0,0 +1,62 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import { vi } from 'vitest'
+
+import SearchInput from '~/components/search-input/SearchInput'
+
+const searchMock = ''
+const setSearchMock = vi.fn()
+
+describe('SearchInput tests', () => {
+
+  it('should render text correctly', () => {
+    render(<SearchInput search={ searchMock } setSearch={ setSearchMock } />)
```

Comment:
move render to beforeEach

Reply:
@MickhaelSh , i think there is no need, cause last test case uses destructed rerender method from render

Reply:
u can get it from beforeEach too

Reply:
Yes, I`m using rerender in the last test case and because of that I didn`t move render to beforeEach

Reply:
you can get it just assign it to the outside variable, but I see it is even no needed in that test case